components/calendar/src/iso.rs

```diff
@@ -382,13 +388,16 @@ impl Iso {
     //
     // Lisp code reference: https://github.com/EdReingold/calendar-code2/blob/1ee51ecfaae6f856b0d7de3e36e9042100b4f424/calendar.l#L1167-L1189
     pub(crate) fn fixed_from_iso(date: IsoDateInner) -> i32 {
+        let year_saturation = (date.0.year as i64) - 1;
```

Comment:
Nit: Since you're not saturating this variable any more, let's name it differently

```suggestion
        let prev_year = (date.0.year as i64) - 1;
```